backend/addcorpus/save_corpus.py

```diff
@@ -48,7 +48,10 @@ def _copy_corpus_attributes(corpus_definition: CorpusDefinition, configuration:
         'word_models_present',
     ]
 
-    defined = get_defined_attributes(corpus_definition, attributes_to_copy)
+    try:
+        defined = get_defined_attributes(corpus_definition, attributes_to_copy)
+    except Exception as e:
+        raise e
```

Comment:
To catch the exceptions happening at lower levels. Otherwise, it would be impossible to tell which of the CorpusDefinition's properties raise the `NotImplementedError`.